formatters/simplecov/simplecov.go

```diff
@@ -42,6 +42,7 @@ func (f *Formatter) Parse() error {
 	}
 	jf, err := os.Open(f.Path)
 	if err != nil {
+		logrus.Debugf("failed to open coverage file %s", f.Path)
```

Comment:
This feels like it's either redundant (because returning `errors.WithStack` should cause useful output somewhere else), or should be a higher level then `Debug`.